config/routes.rb

```diff
@@ -8,6 +8,12 @@
     get "/", to: redirect("/docs/")
   end
 
+  if FeatureService.enabled?(:new_publish)
```

Comment:
Left here as a placeholder/reminder but once we're ready to go live we can remove this guard